website/docs/docs/collaborate/govern/project-dependencies.md

```diff
@@ -23,6 +21,31 @@ This year, dbt Labs is introducing an expanded notion of `dependencies` across m
 - **Packages** &mdash; Familiar and pre-existing type of dependency. You take this dependency by installing the package's full source code (like a software library).
 - **Projects** &mdash; A _new_ way to take a dependency on another project. Using a metadata service that runs behind the scenes, dbt Cloud resolves references on-the-fly to public models defined in other projects. You don't need to parse or run those upstream models yourself. Instead, you treat your dependency on those models as an API that returns a dataset. The maintainer of the public model is responsible for guaranteeing its quality and stability.
 
+
+## Prerequisites
+
+In order to add project dependencies and resolve cross-project `ref`, you must:
+- Have the feature enabled (speak to your account team)
+- Use dbt v1.6 for **both** the upstream ("producer") project and the downstream ("consumer") project.
+- Have a deployment environment in the upstream ("producer") project [that is set to be your production environment](/docs/deploy/deploy-environments#set-as-production-environment-beta)
+- Have a successful run of the upstream ("producer") project
+- Have a multi-tenant or single-tenant [dbt Cloud Enterprise](https://www.getdbt.com/pricing) account -- IS DEPLOYMENT /TENANCY INFO RIGHT?
```

Comment:
> IS DEPLOYMENT /TENANCY INFO RIGHT?

Not yet supported on Azure ST. It is supported everywhere else (including MT + ST). Where do we document that?